models/User.php

```diff
@@ -66,11 +69,44 @@ public function attributeLabels()
         ];
     }
 
+    public function fields()
+    {
+        return [
+            'id',
```

Comment:
@skavt Implement `UserResource` and move the `fields()` implementation there. Remember, in majority of cases API should interact with resource models, not directly the models.